docker/chaos-proxy/Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG ARG_OPENJDK_IMAGE=openjdk:8-jre-alpine
+ARG ARG_BUILD_IMAGE=maven:3.6.1-jdk-8-alpine
```

Comment:
Latest is 3.6.3, although the jdk-8-alpine flavour no longer exists. It's either jdk-8-slim or ibmjava-8-alpine.